packages/astro/src/core/build/internal.ts

```diff
@@ -5,8 +5,11 @@ import { prependForwardSlash } from '../path.js';
 import { viteID } from '../util.js';
 
 export interface BuildInternals {
-	// Pure CSS chunks are chunks that only contain CSS.
-	pureCSSChunks: Set<RenderedChunk>;
```

Comment:
`pureCSSChunks` was previously unused. I've re-purposed it as `cssChunkModuleIds`.